parsers/MD.py

```diff
@@ -280,30 +256,14 @@ def fetch_production(
         field_values = get_data(session)
 
         production = {
-            "solar": None,
-            "wind": None,
-            "biomass": 0.0,
+            "solar": field_values[FIELD_LABEL_to_INDEX["solar"]],
+            "wind": field_values[FIELD_LABEL_to_INDEX["wind"]],
+            "biomass": field_values[FIELD_LABEL_to_INDEX["biomass"]],
             "nuclear": 0.0,
```

Comment:
Could possibly remove this nuclear 0 as well, we generally only want to return the data that are reported by the API, not assign static values.